src/numerics/petsc_matrix.C

```diff
@@ -23,8 +23,10 @@
 
 // Local includes
 #include "libmesh/petsc_matrix.h"
+
```

Comment:
Are you wanting to add a blank line after the header which is named after the current source file? I don't think we've done this in the past, and I prefer not to change it since then the comment `// Local includes` does not seem to apply to all the other headers.

Reply:
I like making it clear that "this one header isn't in the same category as all the others for a reason", vs "we can't *quite* get alphabetical order right.

I should have added another comment, though; what we've done in some other .C files (e.g. dof_map.C),  is `// Local includes` for the source-file-specific header and `// libMesh includes` for the rest.  I'll fix that now and see if that kick resolves the file handle issues too.